QChem.py

```diff
@@ -290,41 +294,54 @@ def __init__(self, config):
         super().__init__()# necessary for every derived class of QCMethod
         # hooks as {function_name : hook_string}
         self.cfg = config
-        self.hooks = {'molecule' : '$molecule',
+        self.hooks = {'elconf' : '$molecule',
+                      'frag_xyz': '$molecule'
         }
 
-
-    @var_tag(V.molecule)
-    def molecule(self, i, data):
+    @var_tag(V.elconf)
+    def elconf(self, i, data):
+        """ Get molecular/fragment configuration """
+        mLogger.info("input elconf(s)", extra={"Parsed" : V.elconf})
```

Comment:
In the logging string it's better to write out `elconf` as `Charge/Multiplicity`.